### PR TITLE
refactor: fund L2 accounts with ETH

### PR DESCRIPTION
**Pull request by platocrat**
**Description**
Fund L2 accounts with Georgios' [script](https://github.com/ethereum-optimism/optimism/blob/master/integration-tests/test/shared/env.ts)

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -8,7 +8,19 @@ module.exports = {
       accounts: {
         mnemonic: 'test test test test test test test test test test test junk'
       }
-    }
+    },
+    // Add this network to your config!
+    optimism: {
```
Comment (smartcontracts) [inline hardhat.config.js:13]: Shouldn't be included in the initial config

**Review by smartcontracts — CHANGES_REQUESTED** [review]
I think we might want to hold off on this PR for now. We should come up with a standardized method for this stuff. Not sure if the stuff that georgios hacked together is the ideal, but I haven't looked too hard at it. I just don't want to add too much complexity to this tutorial if it's avoidable.

File: hardhat.config.js (hunk 1 above)
Comment (platocrat) [reply hardhat.config.js:13]: I included it so the team could help me debug 😔

**Comment by smartcontracts** [comment]
Closing since the hardhat accounts are now automatically funded

**Pull request closed without merging**